docs/installation/launching-ovn-kubernetes-on-kind.md

```diff
@@ -53,6 +54,13 @@ $ ./kind.sh
 $ popd
 ```
 
+**NOTE:** If you run into issues with installing jinjanate on Ubuntu due to [PEP-0668](https://peps.python.org/pep-0668/) you can work around via:
```

Comment:
Considering this is a PEP issue, I am wondering if the correct way to fix this is to require pipx as prerequisite and fix kind.sh to use pipx instead.